src/main/resources/swagger/api-docs.yaml

```diff
@@ -1425,6 +1425,49 @@ paths:
           description: Method Configuration failed to resolve input expressions with the supplied Entity
         500:
           description: Internal Error
+  '/api/libraries':
```

Comment:
we're trying to alphabetize the swagger so this should be further up (line 307ish)